src/dosbox.cpp

```diff
@@ -796,6 +796,20 @@ void DOSBOX_Init()
 	                  "On 'auto' the mode is determined by 'sbtype'.\n"
 	                  "All OPL modes are AdLib-compatible, except for 'cms'.");
 
+	pstring = secprop->Add_string("opl_fadeout", when_idle, "off");
+	pstring->Set_help(
+	        "Fadeout the OPL synth output after the last IO port write:\n"
+	        "  off:       Don't fadeout; residual output will play forever (default).\n"
+	        "  on:        Wait 350 ms before fading out over a 350 ms period.\n"
```

Comment:
Dunno, isn't the wait a bit short? Maybe 500 would be safer?

Reply:
Sure; still reasonable and not too onerous if you need to hear the fixed tone.